worker/include/Channel/UnixStreamSocket.hpp

```diff
@@ -38,6 +38,19 @@ namespace Channel
 		// Others.
 		size_t msgStart{ 0 }; // Where the latest message starts.
 	};
+
+	class ChannelWrapper
+	{
+	public:
+		explicit ChannelWrapper(int consumerFd, int producerFd)
```

Comment:
IMHO this must not be `explicit` (no need for it since it has two arguments).

Reply:
already removed.